assessments/template.md

```diff
@@ -1,8 +1,7 @@
 # Assessment template
 
-Prepared by: `Name`
-Date: `Date`
-
+Prepared by: \<name> ([@add-link-to-your-github-id](https://github.com/cncf/techdocs))<br>
+Date: 2021-mm-dd
```

Comment:
This close to the end of 2021, maybe we should make this `yyyy-mm-dd`?

Reply:
I'd vote to keep this as is and updated it in January. Then it'll be good for a full year! 😄

Reply:
Can we open timed issues? Set an alarm for ourselves? :smile: